proto/cosmos/staking/v1beta1/tx.proto

```diff
@@ -199,3 +233,93 @@ message MsgUpdateParams {
 //
 // Since: cosmos-sdk 0.47
 message MsgUpdateParamsResponse {};
+
```

Comment:
Shouldn't all these new messages have the "cosmos.msg.v1.signer" and the "amino.name" defined? For example
```protobuf
message MsgUnbondValidator {
  option (cosmos.msg.v1.signer) = "validator_address";
  option (amino.name)           = "cosmos-sdk/x/staking/MsgUnbondValidator";
  string validator_address = 1 [(gogoproto.moretags) = "yaml:\"address\""];
}
```